### DISCUSSION TITLE
Generating Theta Waves

### DISCUSSION THREAD

**Discussion by shatton412** [jonescompneurolab/hnn-core#666] [General]
Does anyone have any ideas on how to generate theta waves in this model? I have a raw EEG signal that has high gamma and theta powers and am trying to simulate these in the HNN.

**Comment by rythorpe** [comment]
I'd start with our [alpha tutorial](https://jonescompneurolab.github.io/hnn-core/stable/auto_examples/workflows/plot_simulate_alpha.html#sphx-glr-auto-examples-workflows-plot-simulate-alpha-py) and just decrease the `burst_rate` parameter in the `net.add_bursty_drive()` method.
